jwst/pipeline/calwebb_coron3.cfg

```diff
@@ -7,3 +7,4 @@ class = "jwst.pipeline.Coron3Pipeline"
       [[klip]]
       [[outlier_detection]]
       [[resample]]
+        blendheaders = false
```

Comment:
Should this also be set in `calwebb_coron3.py` `Coron3Pipeline .spec` class attribute?

Reply:
I would say yes, otherwise if someone runs via the `Coron3Pipeline` class, without the config file, it won't work right.

Reply:
As far as I know, hierarchical definitions in `spec` cannot be done. `spec` defines parameters for the specific class; it cannot affect other things.

This, along with the nirspec lamp processing and possible others, REQUIRE some sort of config file setup, at the moment, to operate correctly. This is due to both the above situation with `spec`, and the fact that CRDS-retrieved parameter files have not been implemented. Also, the design discussion of "are pipelines in full control" vs "pipelines a thin veneer over" their substeps, need some resolution. Hearken back to the very recent discussion of `save_results`.

This current implementation is "baseline" with what has been implemented. Options are:

1. Leave as is
2. Hardwire `blendheaders` into the code
3. Go ahead and create/ingest the necessary CRDS step parameter file for coron3.

My initial impulse was option 2.

Note: If set wrong, science itself is not muddled. The results keywords will be correct. Just the HDRTAB would be unnecessarily repeated, with the wrong filenames.

Discuss!

Reply:
So if it doesn't work to set `blendheaders = False` for the `resample` step in the `spec` definition, can it at least be set somewhere in the `Coron3Pipeline` process method itself, like we do with `save_results` settings for some steps in pipeline modules?

Reply:
If we are going with option 2, then setting it by hand, outside of `spec`, is probably the consistent (if not elegant) solution.  Somewhere at the top of process:

```
self.resample.blendheaders = False
```

A user won't be able to modify it, but we don't actually want them to be able to.

Reply:
And then you can remove it from the `.cfg` file, as it will be redundant.

Reply:
Consensus is to hardwire. Done.